addons/binding/org.openhab.binding.blebox/src/main/java/org/openhab/binding/blebox/handler/DimmerHandler.java

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.blebox.handler;
+
+import static org.openhab.binding.blebox.BleboxBindingConstants.CHANNEL_BRIGHTNESS;
+
+import java.math.BigDecimal;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+import org.eclipse.smarthome.core.library.types.OnOffType;
+import org.eclipse.smarthome.core.library.types.PercentType;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.binding.BaseThingHandler;
+import org.eclipse.smarthome.core.types.Command;
+import org.openhab.binding.blebox.devices.Dimmer;
+import org.openhab.binding.blebox.internal.BleboxDeviceConfiguration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link DimmerHandler} is responsible for handling commands, which are
+ * sent to one of the channels.
+ *
+ * @author Szymon Tokarski - Initial contribution
+ */
+public class DimmerHandler extends BaseThingHandler {
+
+    private Logger logger = LoggerFactory.getLogger(DimmerHandler.class);
+    private Dimmer dimmer;
+
+    Runnable runnable = new Runnable() {
```

Comment:
These runnable variables in all the handlers all seem to follow a generic pattern. You might try to crate a generic variant so you don't have to copy-paste large parts of this code. Especially when something generic would change you now need to make sure you wont' forget a handler implementation.